src/org/usfirst/frc4904/robot/commands/StartOuttakeMotors.java

```diff
@@ -0,0 +1,51 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class StartOuttakeMotors extends Command {
+	
+	public StartOuttakeMotors() {
+		// Use requires() here to declare subsystem dependencies
+		requires(RobotMap.Component.ballIO);
+	}
+	
+	// Called just before this Command runs the first time
+	@Override
+	protected void initialize() {
```

Comment:
This will work fine, but for convention's sake, I would suggest changing this entire command to a `CommandGroup`. Then, in the constructor, you can `addParallel` three `MotorConstant` commands.